umbral/keys.py

```diff
@@ -125,6 +145,14 @@ def secret_key_by_label(self, label: bytes) -> SecretKey:
 
```

Comment:
Not related to this OR but we discussed this extra hoop before in discord (https://discord.com/channels/411401661714792449/411401661714792451/860193967923527730)

Reply:
Yep, I remember - see my concerns in https://github.com/nucypher/rust-umbral/pull/64